src/scss/dp/overrides/components/_footer.scss

```diff
@@ -0,0 +1,54 @@
+.footer {
+  background-color: $color-banner-bg;
```

Comment:
This should be set to `$color-grey-100` in order to match the footer bg colour across the website. The text colour should then be set to `$color-grey-5`